src/modules/mount/main.py

```diff
@@ -149,6 +149,21 @@ def get_btrfs_subvolumes(partitions):
 
     return btrfs_subvolumes
 
+def enable_swap_partition():
+    swap_is_exist=subprocess.check_output(["swapon","-s"], universal_newlines=True)
+    if ( swap_is_exist == "" ):
+        all_partitions=subprocess.check_output(["lsblk", "-l", "-f", "-n", "-p"], universal_newlines=True).split("\n")
+        for p in all_partitions:
+            q = p.split()
+            if ( len(q) > 1 and q[1] == "swap" ):
+                swap = q[0]
+                try:
+                    subprocess.check_call(["swapon", swap])
+                except Exception as e:
+                    try:
+                        subprocess.check_call(["mkswap", swap])
```

Comment:
We shouldn't blindly call `mkswap` here.  `mkswap` should have already been run by the partition module.